.github/issue_template.md

```diff
@@ -1,9 +1,25 @@
-_The template below is mostly useful for bug reports and support questions. Feel free to remove anything which doesn't apply to you and add more information where it makes sense._
 ---
+name: Device Plugin Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: elezar, kklues
```

Comment:
I don't know whether I want to set this here explicitly.

Not a blocker though.

Reply:
you mean, rm that line?